### DISCUSSION TITLE
Welcome to MeteoInfo Discussions!

### DISCUSSION THREAD

**Discussion by Yaqiang** [meteoinfo/MeteoInfo#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Shahidlatifbhutto** [comment]
Chinese province in the cn_Province.dbf file shows these names. However names should be in English.

▒▒╛⌐╩╨
--
╠∞╜≥╩╨
║╙▒▒╩í
╔╜╬≈╩í
─┌├╔╣┼╫╘╓╬╟°
┴╔─■╩í
╝¬┴╓╩í
║┌┴·╜¡╩í
╔╧║ú╩╨
╜¡╦╒╩í
╒π╜¡╩í
░▓╗╒╩í
╕ú╜¿╩í
╜¡╬≈╩í
╔╜╢½╩í
║╙─╧╩í
║■▒▒╩í
║■─╧╩í
╣π╢½╩í
╣π╬≈╫│╫σ╫╘╓╬╟°
║ú─╧╩í
╓╪╟∞╩╨
╦─┤¿╩í
╣≤╓▌╩í
╘╞─╧╩í
╬≈▓╪╫╘╓╬╟°
╔┬╬≈╩í
╕╩╦α╩í
╟α║ú╩í
─■╧─╗╪╫σ╫╘╓╬╟°
╨┬╜«╬¼╬ß╢√╫╘╓╬╟°
╠¿═σ╩í
╧π╕█╠╪▒≡╨╨╒■╟°
░─├┼╠╪▒≡╨╨╒■╟°
╓╨╣·
╓╨│»╣▓╙╨



</body>

</html>

**Reply by Yaqiang** [reply]
The names are Chinese characters.

**Reply by Shahidlatifbhutto** [reply]
<img width="816" alt="f479b7a4e6483755c1c387455df9bb2" src="https://github.com/user-attachments/assets/35157288-a266-4577-abb8-287e2f841c05" />
Sir, how do I convert these Chinese names into English names? After changing fonts, I finally found these Chinese names, but I want to show their English names.

**Reply by Shahidlatifbhutto** [reply]
<img width="1019" alt="75c2a0b7d5c876741b8f6f1c7ed4caf" src="https://github.com/user-attachments/assets/0c22a4a5-9e83-4f2c-a26e-17fd0e138ddd" />
